app/controllers/basespace_controller.rb

```diff
@@ -44,30 +39,16 @@ def projects
       return
     end
 
-    # 1024 is the maximum limit allowed by Basespace.
-    # If users reach this limit, we will need to implement multiple requests to fetch all the projects.
-    response = HttpHelper.get_json(
-      BASESPACE_CURRENT_PROJECTS_URL,
-      { limit: BASESPACE_PAGE_SIZE },
-      "Authorization" => "Bearer #{access_token}"
-    )
+    projects = basespace_projects(access_token)
```

Comment:
to avoid confusion with idseq projects, I recommend basespace projects prefixed everywhere they appear, for example, `bs_projects = basespace_projects(access_token)`